### ISSUE TITLE
Cannot find /mnt/data/wy/model/BiomedCLIP-PubMedBert_256-vit_base_path16_224 on HunggingFace

### ISSUE THREAD

**Issue by Kingofolk** [big-white-rabbit/Coarse-to-Fine-Grained-Contrastive-Learning#1]
Hi big-white-rabbit,

Thanks a lot for your great work! I have trouble in find the pretrained model BiomedCLIP-PubMedBert_256-vit_base_path16_224 in HunggingFace, should this be renamed as microsoft/BiomedNLP-BiomedBERT-base-uncased-abstract-fulltext?

Meanwhile, I v come across the dimension mismatch in the following line:
_global_loss = gloria.global_loss(image_feat, torch.mean(keywords_embeds, dim=1), gt_matrix)_

The errors are:
_RuntimeError: Expected size for first two dimensions of batch2 tensor to be: [1, 1024] but got: [1, 768]._

I v checked the dimension, the image_feat with a feature dimension of 1024, but the keywords_embeds is 768. Should I change the attn_dimension of image_feat to 768 or this is caused by the wrong text encoder?

Best regards
Kingofolk
